backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/textfield/internal/BpkTextFieldImpl.kt

```diff
@@ -86,6 +86,7 @@ internal fun BpkTextFieldImpl(
     minLines: Int = 1,
     maxLines: Int = 1,
     interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
+    isFocused: Boolean? = null,
```

Comment:
Why do we need to add this field? Can we always use the mutable interaction source instead?

Reply:
We found it caused issues with the screen reader when changing the focus state programatically.

## Before

https://github.com/user-attachments/assets/93455770-979f-4c68-877d-b7ddbe5abb10

## After
https://github.com/user-attachments/assets/acdd051d-7778-4390-af0e-51626b8c639e